app/Http/Controllers/ImagesController.php

```diff
@@ -58,4 +61,30 @@ public function show(Post $post, Request $request)
 
         return $server->getImageResponse($post->getMediaPath(), $request->all());
     }
+
+    /**
+     * Edits and overwrites an image based on given request parameters.
+     *
+     * @param  $id
+     * @param  Request $request
+     * @return \Illuminate\Http\Response
+     */
+    public function update($id, Request $request)
+    {
+        $post = Post::findOrFail($id);
+        $originalImage = $post->getMediaUrl();
+
+        // Only supports rotation, right now.
+        if ($request->input('rotate')) {
+            $value = (int) -$request->input('rotate');
+
+            $originalImage = Image::make($originalImage)->rotate($value)->encode('jpg', 75);
+        }
+
+        $image = $this->aws->storeImageData((string) $originalImage, 'edited_' . $post->id);
```

Comment:
Is this also rotating the original image? The `edited_` files only make sense to keep around until we turn on Glide.

Reply:
oh it doesn't! But I think I am still confused about the difference between the two. I thought the `edited_` images were the smaller, cropped images that we should be using for display and returning in the API and the original images we shouldn't even work with.

Reply:
The `edited_` images were just a "quick fix" to get cropped images until Glide was ready to ship, and so once Glide is enabled everywhere we can just delete them. We want to use the original source as the base image for those future Glide-powered crops, since it gives us the highest quality material in case we ever want larger formatted images.

Reply:
So for now, let's rotate both – the "user facing" `edited_` images and the "source of truth" original versions.

Reply:
AH yes yes now i remember. Will do, thanks!